src/components/UploadCSVView/EditableRow/EditableRow.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Input,
+  InputGroup,
+  InputRightElement,
+  Checkbox,
+  FormControl,
+  FormErrorMessage,
+} from '@chakra-ui/react';
+import { WarningIcon } from '@chakra-ui/icons';
+import './EditableRow.css';
+import 'react-datepicker/dist/react-datepicker.css';
+import { useForm, Controller } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import DatePicker from 'react-datepicker';
+import BoxSchema from '../../UploadCSV/UploadCSVUtils';
+// import { formatDate } from '../../../common/utils';
+import CheckIcon from '../../../assets/check.png';
+
+const EditableRow = ({ editFormData, handleEditFormSubmit }) => {
+  const {
+    register,
+    control,
+    handleSubmit,
+    formState: { errors },
+  } = useForm({
+    resolver: yupResolver(BoxSchema),
+    defaultValues: {
+      date: editFormData.date,
+      boxNumber: editFormData.boxNumber,
+      zipCode: editFormData.zipCode,
+      launchedOrganically: editFormData.launchedOrganically,
+    },
+    delayError: 750,
+  });
+
+  const onSave = editRowData => {
+    handleEditFormSubmit(editRowData);
+  };
+
+  return (
+    <tr className="edit-row" key={editFormData.id}>
```

Comment:
Same here, can we use Chakra tables - https://chakra-ui.com/guides/integrations/with-react-table?